Sources/OktaIdx/Extensions/DeprecationWarnings.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2022-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+
+@available (*, unavailable, renamed: "InteractionCodeFlow")
+public struct IDXClient {
+    @available (*, unavailable)
+    public struct Configuration {
+        @available (*, unavailable)
+        public init(issuer: String,
+                    clientId: String,
+                    clientSecret: String?,
+                    scopes: [String],
+                    redirectUri: String) {}
+    }
+    
+    @available (*, unavailable)
+    public enum Option: Hashable {}
+    
+    @available (*, unavailable,
```

Comment:
Do you think it would be helpful if we include the version we deprecated these and introduced the  InteractionCodeFlow?

Reply:
that sounds good.